pimd/pim6_mld.c

```diff
@@ -2423,6 +2423,7 @@ static void gm_show_if_one(struct vty *vty, struct interface *ifp,
 	querier = IPV6_ADDR_SAME(&gm_ifp->querier, &pim_ifp->ll_lowest);
 
 	if (js_if) {
+		json_object_string_addf(js_if, "name", "%s", ifp->name);
```

Comment:
Just `json_object_string_add` not enough here? IMO, formatting is not needed here.

Reply:
ACK